contracts/strategies/uniswap/UniswapV3ConverterStrategyLogicLib.sol

```diff
@@ -717,6 +717,10 @@ library UniswapV3ConverterStrategyLogicLib {
     uint covered;
     if (loss > 0) {
       covered = UniswapV3DebtLib.coverLossFromRewards(loss, state.strategyProfitHolder, vars.tokenA, vars.tokenB, address(vars.pool));
+      uint notCovered = loss - covered;
+      if (notCovered > 0) {
```

Comment:
Using of (notCovered != 0) would be more efficient here from gas point of view, see https://forum.openzeppelin.com/t/a-collection-of-gas-optimisation-tricks/19966